generation/spade_gan/spade_gan.ipynb

```diff
@@ -0,0 +1,2665 @@
+{
```

Comment:
<p>Could you please help move this cell after Setup imports? Keep it in line with other tutorials.</p><p>Refer to:</p><p><a href="https://github.com/Project-MONAI/tutorials/blob/main/3d_segmentation/spleen_segmentation_3d.ipynb" target="_blank">https://github.com/Project-MONAI/tutorials/blob/main/3d_segmentation/spleen_segmentation_3d.ipynb</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1794/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>